[CS2103T-T11-3] AddMin+

DivineDX opened the pull request:
@shihaoyap @Crisgu @calvincxz @eshwarkp

charliechoong commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,88 +295,172 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *` |Admin Staff|find upcoming events | plan on the logistics needed ahead of time
+
+|`* *` |Manager|tag my employees with comments/remarks | have better manpower allocation for events
```
User stories are comprehensive, insightful and revelatory. Well done!

charliechoong commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,88 +295,172 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 +
 Use case resumes at step 2.
 
+
```
MSS are clear and concise.

charliechoong commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,88 +295,172 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  Program should run smoothly and without bugs
+.  Should be able to work independent of a remote server, database, and internet connection
 
```
for line 454, should start with "Should..." to standardize with other points.

charliechoong reviewed (commented):
Good luck with tP.

arjavibahety commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,88 +295,172 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+* 2.The Employee_ID/Event_ID is invalid.
++
+[none]
+** 2a.Addmin+ shows an error message.
```
We think it should be AddMin+ and not Addmin+

arjavibahety commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,88 +295,172 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+*Extensions*
+
+* 2.The Employee_ID/Event_ID is invalid.
```
The number should be 3a 
The number for the extension for the last 3 use cases is not correct
The spacing after the number is not consistent

arjavibahety commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,88 +295,172 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+1.  User requests to view the company’s current list of employees and events.
+2.  AddMin+ shows the list of employees and events.
+3.  User states an Employee_ID that is matched to an Event_ID
```
It would be better if you could be more consistent with regards to using full stops at the end of sentences

arjavibahety reviewed (commented):
Good job with the developer guide 👍 
There are some inconsistencies which we have highlighted

-Team T13-2

arjavibahety commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,88 +295,172 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
-** 3a1. AddressBook shows an error message.
+** 2a.Addmin+ shows an error message.
 +
 Use case resumes at step 2.
```
This should be Use case ends

ryantay232 commented on docs/DeveloperGuide.adoc:
```diff
@@ -92,155 +93,297 @@ The `UI` component,
-* `VersionedAddressBook#commit()` -- Saves the current address book state in its history.
-* `VersionedAddressBook#undo()` -- Restores the previous address book state from its history.
-* `VersionedAddressBook#redo()` -- Restores a previously undone address book state from its history.
+Step 1. The user launches the application for the first time and generates statistics via the "statistics /g" command.
```
Not formatted properly. Below is formatted properly.

ryantay232 commented on docs/DeveloperGuide.adoc:
```diff
@@ -92,155 +93,297 @@ The `UI` component,
+demand (number of employees required by event, generated in step 3).
 [NOTE]
-If the `currentStatePointer` is at index `addressBookStateList.size() - 1`, pointing to the latest address book state, then there are no undone address book states to restore. The `redo` command uses `Model#canRedoAddressBook()` to check if this is the case. If so, it will return an error to the user rather than attempting to perform the redo.
+If supply < demand, throws exception to user. If supply > demand, employees will be randomly selected instead.
```
Can't really understand this line. The angled brackets used here confused me.

ryantay232 commented on docs/DeveloperGuide.adoc:
```diff
@@ -92,155 +93,297 @@ The `UI` component,
 
-Step 6. The user executes `clear`, which calls `Model#commitAddressBook()`. Since the `currentStatePointer` is not pointing at the end of the `addressBookStateList`, all address book states after the `currentStatePointer` will be purged. We designed it this way because it no longer makes sense to redo the `add n/David ...` command. This is the behavior that most modern desktop applications follow ...
+._Sequence Diagram of the AutoAllocate Command_
+image::AutoAllocateSequenceDiagram.png[]
```
What is this "a" that been returned by AddressBookParser and AutoAllocateCommandParser?

lumos309 replied:
"a" is specified as the AutoAllocateCommand in the diagram.

ryantay232 reviewed (commented):
Looks good!

lumos309 commented on docs/DeveloperGuide.adoc:
```diff
@@ -92,155 +93,297 @@ The `UI` component,
 
-image::UndoRedoState3.png[]
+._Program flow of the Auto Allocate Feature_
+image::AutoAllocateFlowChart.png[]
```
Start and end circles in diagram missing. "Done" step at the end is unnecessary. "Ensure no syntax error" is a lower-level detail than the rest of the steps.

lumos309 commented on docs/DeveloperGuide.adoc:
```diff
@@ -92,155 +93,297 @@ The `UI` component,
 
-Step 6. The user executes `clear`, which calls `Model#commitAddressBook()`. Since the `currentStatePointer` is not pointing at the end of the `addressBookStateList`, all address book states after the `currentStatePointer` will be purged. We designed it this way because it no longer makes sense to redo the `add n/David ...` command. This is the behavior that most modern desktop applications follow ...
+._Sequence Diagram of the AutoAllocate Command_
+image::AutoAllocateSequenceDiagram.png[]
```
The isAvailableForEvent() method is called multiple times, once for each employee. Shouldn't it be displayed in some sort of loop?

lumos309 commented on docs/DeveloperGuide.adoc:
```diff
@@ -92,155 +93,297 @@ The `UI` component,
 
-image::CommitActivityDiagram.png[]
+._Activity Diagram of the AutoAllocate Command_
+image::AutoAllocateActivityDiagram.png[]
```
What does it mean to "convert" to unique Id? A clearer explanation might be better.

What happens if the command is not successful?

lumos309 commented on docs/DeveloperGuide.adoc:
```diff
@@ -92,155 +93,297 @@ The `UI` component,
+
+Step 2. The user executes `add_ev n/Free Cofee ...` to add a new event into the list. The distinctDatesList will be flushed.
+
+image::DisplayScheduleState1.png[]
```
What does "flushing" the distinctDatesList mean? It's not clear how the list still contains the existing events after the flush.

lumos309 commented on docs/DeveloperGuide.adoc:
```diff
@@ -58,28 +58,29 @@ Each of the four components
 
-.Structure of the UI Component
+._Structure of the UI Component_
 image::UiClassDiagram.png[]
```
Should ScheduleBox point to {abstract} Tabs with a hollow triangle instead of an arrow, since it extends the abstract class?

lumos309 commented on docs/DeveloperGuide.adoc:
```diff
@@ -92,155 +93,297 @@ The `UI` component,
+
+Step 1. The user launches the application for the first time. The `distinctDatesList` will be initialized based on the initial event book state.
+
+image::DisplayScheduleState0.png[]
```
Consider referring to the course material on how a class diagram should be drawn.

Commit:
Changes to fit checkStyle.

shihaoyap committed:
fixed testcase for schedule and event related commands

shihaoyap committed:
Fix code

Commit:
Added upcoming event statistic.
Changes for Statistics UI.

DivineDX committed:
Upgrade EditEvent Command

Commit:
Merge pull request #234 from AY1920S1-CS2103T-T11-3/stats

Stats Branch

DivineDX committed:
Merge branch 'master' into EventDateTime

shihaoyap committed:
Added DistinctDate object testing and generator
Added test case for generate_schedule command

shihaoyap committed:
Updated `generate_schedule` ug command

DivineDX committed:
Merge pull request #235 from AY1920S1-CS2103T-T11-3/EventDateTime

EventDateTime Upgrades and Improvements

shihaoyap committed:
Merge branch 'master' into Shihao_Branch

Crisgu committed:
lack hour

shihaoyap committed:
Improved code quality

shihaoyap committed:
Merge pull request #233 from AY1920S1-CS2103T-T11-3/Shihao_Branch

1) Setup typical event and eventBuilder test structure

DivineDX committed:
Add EmployeeCheck Method

shihaoyap committed:
Merge branch 'Cris-Branch' of https://github.com/AY1920S1-CS2103T-T11-3/main into Cris-Branch

# Conflicts:
#	src/main/resources/view/MainWindow.fxml

Commit:
Statistic UI changes

Crisgu committed:
use event list

Crisgu committed:
new

DivineDX committed:
Fix Bug: EventDateTimeMapping

Crisgu committed:
Finish up Pay and UndoPay

Commit:
Statistics UI updated to display upcoming events list.

Commit:
Checkstyle fix.

Commit:
Merge branch 'master' into stats

Commit:
Added Distinct Tags Statistic (Overall & Individual)

Commit:
Merge branch 'stats' of https://github.com/AY1920S1-CS2103T-T11-3/main into stats

Commit:
Fix Checkstyle

DivineDX committed:
Fix: Bugs & Code Quality Improvements

DivineDX committed:
Restore Test Cases

Crisgu committed:
Edit javadoc

DivineDX committed:
Merge pull request #238 from AY1920S1-CS2103T-T11-3/Cris-FinalFix

Cris final fix

Crisgu committed:
user guid update

shihaoyap committed:
minor edits to some of cris commands / Ui

DivineDX committed:
Merge branch 'master' into EventDateTime

DivineDX committed:
Merge pull request #239 from AY1920S1-CS2103T-T11-3/Shihao_Branch

minor edits to some of cris commands / Ui

DivineDX committed:
Add ParameterConstraints to UG

shihaoyap committed:
Minor UG, PPP changes

DivineDX committed:
Add Constraint Crossreference to UG

DivineDX committed:
Merge pull request #241 from AY1920S1-CS2103T-T11-3/Shihao_Branch

Minor UG, PPP changes

DivineDX committed:
Checkstyle Fixes

DivineDX committed:
Merge branch 'master' into EventDateTime

DivineDX committed:
Delete Redundant SetAppDate Command

DivineDX committed:
Merge pull request #242 from AY1920S1-CS2103T-T11-3/EventDateTime

Merge PR

Commit:
Major revamp to statistic functionality
- Added pie charts to represent employee and event tags
- List to display upcoming events that still require manpower
- Added detailed statistics to provide the user more information

Commit:
Merge branch 'master' into stats

Commit:
Checkstyle fix

DivineDX committed:
Update EditEvent Logic

Commit:
Merge pull request #237 from AY1920S1-CS2103T-T11-3/stats

Stats

DivineDX committed:
Merge branch 'master' into EventDateTime

DivineDX committed:
Merge pull request #244 from AY1920S1-CS2103T-T11-3/EventDateTime

EditEvent Logic Update

Commit:
Update Statistics UI

Commit:
Merge branch 'master' into stats

Commit:
Update UserGuide.adoc

Commit:
Modified upcoming events list
Modified pie chart data

Commit:
Merge branch 'stats' of https://github.com/AY1920S1-CS2103T-T11-3/main into stats

Commit:
Update Statistic Commands

Commit:
Statistics UI update

Commit:
Statistics Section Images

Commit:
Update UserGuide.adoc

Updated Statistics Commands 
- Functionality description
- Step by step usage with UI examples

shihaoyap committed:
UI fix

Commit:
Fixed issue with list view

Commit:
Statistics UI changes

Commit:
Checkstyle fix

DivineDX committed:
Improve Code Quality: model.Event

calvincxz committed:
improve quality

calvincxz committed:
removed employee joinDate field

DivineDX committed:
Improve Code Quality: Event Predicates

DivineDX committed:
Improve Code Quality: EventCommands & EventParsers

DivineDX committed:
Merge pull request #248 from AY1920S1-CS2103T-T11-3/calvin_new

removed employeeJoinDate field, updated PPP

DivineDX committed:
Merge branch 'master' into stats

shihaoyap committed:
Fixed UI sizing issue. Updated Testcases for Schedule Related commands. Improve Code Quality by abstracting the List recongnition according to which Tab user is on.

shihaoyap committed:
Merge branch 'master' into Shihao_Branch

DivineDX committed:
Merge pull request #246 from AY1920S1-CS2103T-T11-3/stats

Update Statistics UI

DivineDX committed:
Merge branch 'master' into DXEnhancements

shihaoyap committed:
Merge branch 'master' into Shihao_Branch

shihaoyap committed:
Merge pull request #249 from AY1920S1-CS2103T-T11-3/Shihao_Branch

Fixed UI sizing issue. Updated Testcases for Schedule Related command…

DivineDX committed:
Add TestCases for Event

DivineDX committed:
Merge conflict, Fix Checkstyle

Crisgu committed:
rename

Crisgu committed:
undate UG

Crisgu committed:
Merge branch 'master' into Cris-FinalFix

Crisgu committed:
Merge pull request #240 from AY1920S1-CS2103T-T11-3/Cris-FinalFix

user guid update

DivineDX committed:
Edit according to PR Review

DivineDX committed:
Merge branch 'master' into DXEnhancements

DivineDX committed:
Merge pull request #250 from AY1920S1-CS2103T-T11-3/DXEnhancements

CodeQuality & TestCases

Crisgu committed:
UG Update

Crisgu committed:
Merge branch 'master' into Cris-FinalFix

DivineDX committed:
Merge pull request #251 from AY1920S1-CS2103T-T11-3/Cris-FinalFix

UG Update

calvincxz committed:
added allocatem by id. fix some bugs

calvincxz committed:
added unfetch, focus remains on main after fetch

calvincxz committed:
fix employeewindow

calvincxz committed:
fix

Commit:
Change in Detailed Statistics Window.

Crisgu committed:
de-allocated employee payment

Crisgu committed:
de_allocated

Crisgu committed:
de_allocated employee

DivineDX committed:
Merge pull request #252 from AY1920S1-CS2103T-T11-3/calvin_new

more CLI

DivineDX committed:
Merge branch 'master' into Cris-FinalFix

Commit:
Merge branch 'master' into stats

DivineDX committed:
Merge pull request #254 from AY1920S1-CS2103T-T11-3/Cris-FinalFix

Cris final fix

shihaoyap committed:
Merge branch 'master' into stats

Commit:
Merge pull request #255 from AY1920S1-CS2103T-T11-3/stats

Change in Detailed Statistics Window.

Crisgu committed:
UG update

shihaoyap committed:
Added command restriction to specific tabs -
Main Tab: all Commands can work
Schedule Tab: block all employee commands, block allocatem by index
Finance Tab: block all event commands and all allocate commands
Statistics Tab. Block all employee commands, block allocatem by index

shihaoyap committed:
checkstyle

shihaoyap committed:
Updates to commands to increase code quality

shihaoyap committed:
Merge pull request #265 from AY1920S1-CS2103T-T11-3/Shihao_Branch

Added command restriction to specific tabs -

shihaoyap committed:
Test Case for DisplayScheduleBetweenParser

shihaoyap committed:
update checkstyle

shihaoyap committed:
Merge branch 'master' into Shihao_Branch

calvincxz committed:
fix bugs (#262)

* fix bugs

* update ppp

shihaoyap committed:
Merge branch 'master' into Shihao_Branch

shihaoyap committed:
Merge pull request #267 from AY1920S1-CS2103T-T11-3/Shihao_Branch

Shihao branch

shihaoyap committed:
updated DG

shihaoyap committed:
updated PPP

shihaoyap committed:
Update Ui component of DG.

shihaoyap committed:
update UI component

shihaoyap committed:
Minor update to schedule related commands.

shihaoyap committed:
update documentation and test

shihaoyap committed:
updated testcase

shihaoyap committed:
Merge pull request #268 from AY1920S1-CS2103T-T11-3/Shihao_Branch

updated DG

shihaoyap committed:
PPP, UG, DG Updates

calvincxz committed:
block fetch/allocate on stats/finance. update ppp (#269)

* block fetch/allocate on stats/finance
* disable fetch in stats
* update ug dg ppp

shihaoyap committed:
Merge branch 'master' into Shihao_Branch

shihaoyap committed:
Updated PPP

shihaoyap committed:
Updated PPP

shihaoyap committed:
Merge branch 'Shihao_Branch' of https://github.com/AY1920S1-CS2103T-T11-3/main into Shihao_Branch

DivineDX committed:
Fix EventDateTime Bug

shihaoyap committed:
Updated DG

shihaoyap committed:
Update dg

shihaoyap committed:
Merge pull request #272 from AY1920S1-CS2103T-T11-3/Shihao_Branch

PPP, UG, DG Updates

DivineDX committed:
Add Tests to ParserUtil

DivineDX committed:
Merge branch 'master' into DXEnhancements

shihaoyap committed:
PPP Update

shihaoyap committed:
update PPP

shihaoyap committed:
update author tag

calvincxz committed:
fix bugs (#273)

* update ug dg
* added parser tests
* fix bugs

shihaoyap committed:
Merge branch 'master' into Shihao_Branch

shihaoyap committed:
Merge pull request #274 from AY1920S1-CS2103T-T11-3/Shihao_Branch

PPP Update

DivineDX committed:
Add EventParser Tests

DivineDX committed:
Add EventParser Tests

DivineDX committed:
Merge branch 'master' into DXEnhancements

DivineDX committed:
Fix checkstyle

DivineDX committed:
Merge pull request #275 from AY1920S1-CS2103T-T11-3/DXEnhancements

Create and Enhance TestCases

shihaoyap committed:
Minor ug,dg update

shihaoyap committed:
updates

shihaoyap committed:
Merge branch 'Shihao_Branch' of https://github.com/AY1920S1-CS2103T-T11-3/main into Shihao_Branch

DivineDX committed:
Merge pull request #276 from AY1920S1-CS2103T-T11-3/Shihao_Branch

Minor ug,dg update

shihaoyap committed:
improve code quality

shihaoyap committed:
Merge branch 'Shihao_Branch' of https://github.com/AY1920S1-CS2103T-T11-3/main into Shihao_Branch

shihaoyap committed:
minor updatee

shihaoyap committed:
Merge branch 'Shihao_Branch' of https://github.com/AY1920S1-CS2103T-T11-3/main into Shihao_Branch

DivineDX committed:
Enhance AssignDate Command

DivineDX committed:
Merge pull request #277 from AY1920S1-CS2103T-T11-3/Shihao_Branch

Shihao branch

DivineDX committed:
Merge branch 'master' into DXEnhancements

DivineDX committed:
Fix DateRange bug

DivineDX committed:
Enhance DeleteDateMappingCommand

DivineDX committed:
Update UG: EventDates

DivineDX committed:
Update PPP

calvincxz committed:
update ug

calvincxz committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-3/main into final_calvin_branch

calvincxz committed:
update ug, added test

DivineDX committed:
Update DG and Commands

shihaoyap committed:
Merge pull request #279 from AY1920S1-CS2103T-T11-3/final_calvin_branch

update ppp,ug

shihaoyap committed:
Merge branch 'master' into DXEnhancements

shihaoyap committed:
Merge pull request #278 from AY1920S1-CS2103T-T11-3/DXEnhancements

DateTime Enhancements

shihaoyap committed:
Updates to PPP

shihaoyap committed:
Update to UG

shihaoyap committed:
updated UG pictures

shihaoyap committed:
updated PPP

shihaoyap committed:
Ban Finance commands in Schedule and Stats tab

shihaoyap committed:
Update UG

shihaoyap committed:
UG

shihaoyap committed:
updated dg

shihaoyap committed:
no message

Crisgu committed:
DG PPP

shihaoyap committed:
no message

shihaoyap committed:
Merge pull request #281 from AY1920S1-CS2103T-T11-3/Shihao_Branch

Updates to PPP

shihaoyap committed:
Final Touches

DivineDX committed:
Finalize Documentation Updates

DivineDX committed:
Merge branch 'master' into DXEnhancements

DivineDX committed:
Add fixes for DG & UG

shihaoyap committed:
minor edits

DivineDX committed:
Merge pull request #282 from AY1920S1-CS2103T-T11-3/DXEnhancements

Documentation Enhancements

shihaoyap committed:
Merge branch 'master' into Shihao_Branch

shihaoyap committed:
Merge pull request #283 from AY1920S1-CS2103T-T11-3/Shihao_Branch

Shihao branch

DivineDX committed:
Documentation Edits

shihaoyap committed:
upload pdf

calvincxz committed:
Added some test, (#280)

* update ug
* add test for addcommand, deallocateparser

shihaoyap committed:
updated ug minor edit

shihaoyap committed:
Merge branch 'Shihao_Branch' of https://github.com/AY1920S1-CS2103T-T11-3/main into Shihao_Branch

shihaoyap committed:
upload PPP in pdf and html format

shihaoyap committed:
fix ci

shihaoyap committed:
removed html file

DivineDX committed:
Made DateTimeMapping immutable, Added Tests, Update Docs

DivineDX committed:
Merge branch 'master' into DXEnhancements

DivineDX committed:
Merge pull request #284 from AY1920S1-CS2103T-T11-3/Shihao_Branch

Shihao branch

DivineDX committed:
Merge branch 'master' into DXEnhancements

DivineDX committed:
General Clean-Up

Crisgu committed:
dg

Crisgu committed:
dgn

Commit:
Rectified issues and bugs identified

Commit:
Merge branch 'master' into stats

Crisgu committed:
UGDGPPP

Commit:
Fixed Statistic UI issues

DivineDX committed:
Add EventCommand Test Cases

DivineDX committed:
Merge branch 'DXEnhancements' of https://github.com/AY1920S1-CS2103T-T11-3/main into DXEnhancements

Commit:
Checkstyle Fix

DivineDX committed:
Merge pull request #285 from AY1920S1-CS2103T-T11-3/DXEnhancements

Code Quality, Docs, and TestCases

Crisgu committed:
Doc

Crisgu committed:
Merge branch 'master' into Cris-PPP

shihaoyap committed:
someone accidentally changed my tab behavior

shihaoyap committed:
Merge branch 'master' into Shihao_Branch

Commit:
Stats UG images

Commit:
Update Statistics Section

Commit:
Change picture stats

Commit:
Merge branch 'stats' of https://github.com/AY1920S1-CS2103T-T11-3/main into stats

Commit:
Merge branch 'master' into stats

Crisgu committed:
ugdg

Crisgu committed:
ugdg1

DivineDX committed:
Add Manual Testing to Documentation

DivineDX committed:
Merge branch 'master' into DXEnhancements

Commit:
Merge pull request #286 from AY1920S1-CS2103T-T11-3/stats

Statistics Bug Fixes

DivineDX committed:
Merge branch 'master' into DXEnhancements

DivineDX committed:
Fix checkstyle

DivineDX committed:
Merge branch 'DXEnhancements' of https://github.com/AY1920S1-CS2103T-T11-3/main into DXEnhancements

shihaoyap committed:
update testing docs

shihaoyap committed:
Merge branch 'Shihao_Branch' of https://github.com/AY1920S1-CS2103T-T11-3/main into Shihao_Branch

DivineDX committed:
Merge pull request #289 from AY1920S1-CS2103T-T11-3/DXEnhancements

Documentation Update

Crisgu committed:
ugdg2

shihaoyap committed:
z

shihaoyap committed:
Merge branch 'master' into Shihao_Branch

shihaoyap committed:
Merge pull request #288 from AY1920S1-CS2103T-T11-3/Shihao_Branch

someone accidentally changed my tab behavior

Commit:
Update DG for Stats

Commit:
Merge branch 'master' into stats

calvincxz committed:
dg update (#290)

shihaoyap committed:
Merge branch 'master' into stats

Crisgu committed:
finalppp

Crisgu committed:
Merge branch 'master' into Cris-PPP

Commit:
Update Stats DG

shihaoyap committed:
Merge pull request #291 from AY1920S1-CS2103T-T11-3/stats

DG Update for Stats

Crisgu committed:
Merge branch 'master' into Cris-PPP

Commit:
Stats Change

Commit:
Merge branch 'master' into stats

shihaoyap committed:
Merge pull request #292 from AY1920S1-CS2103T-T11-3/stats

Stats Quick Change

shihaoyap committed:
Merge branch 'master' into Cris-PPP

Crisgu committed:
Merge pull request #287 from AY1920S1-CS2103T-T11-3/Cris-PPP

Cris ppp

DivineDX committed:
Update README.adoc

Add Badges

DivineDX committed:
Empty Commit: Rebuild pages, permission granted from Module Coordinator

DivineDX committed:
Merge: Eshwar's PPP (#296): Permission given by Module Coordinator

* PPP update
